buildSrc/build.gradle

```diff
@@ -12,7 +12,7 @@ configurations {
 dependencies {
     compile gradleApi()
     compile localGroovy()
-    compile group: 'com.google.guava', name: 'guava', version: '19.0'
+    compile group: 'com.google.guava', name: 'guava', version: '18.0'
```

Comment:
I'm not sure why that's there; I had pulled from WPIRoboticsProjects/master so this may have gotten changed